docs/UserGuide.md

```diff
@@ -286,19 +286,24 @@ excuted.
 
 Delete a useless floating task/event/deadline on TaskTracker.
 
-    del "TASK_NAME"
+    del TASK_UNIQUE_INDEX
 
-* Task name should be in a pair of quotation marks.
+* `TASK_UNIQUE_INDEX` will be different from each to each single task.
```

Comment:
Basically, it will start from 0 or (00000), and  with each one task added, the number will increase by 1. It is just a counter in other words. But I think it is not necessary to explain this in UserGuide, and that why I did not explain here.